members/CusSad468cc61669068789.yaml

```diff
@@ -0,0 +1,35 @@
+bioguide: CusSad468cc61669068789
+contact_form:
+  driver: chrome
+  steps:
+    - visit: "https://www.flrules.org/Gateway/View_notice.asp?id=26536986"
+    - find:
+        - selector: input[type='submit'][value='Make Comment']
+    - click_on:
+        - selector: input[type='submit'][value='Make Comment']
+    - fill_in:
+        - name: first name
+          selector: input[name='fname']
+          value: $NAME_FIRST
+          required: false
+        - name: last name
+          selector: input[name='lname']
+          value: $NAME_LAST
+          required: false
+        - name: email
+          selector: input[name='email']
+          value: $EMAIL
+          required: false
```

Comment:
Even though the form isn't showing email required it's rejecting submissions without an email. 
![image](https://user-images.githubusercontent.com/110497520/203416494-15f70e75-5d53-4ab3-94d3-4ee036859557.png)